public/vendor/plugins/autolink/autolink.js

```diff
@@ -1,45 +0,0 @@
-(function () {
```

Comment:
Why was this removed?

Reply:
Good question, wasn't me. Put it back in though.